calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,150 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI.
+
```

Comment:
Added to Concepts section.

Reply:
> ... we recommend that you create a manifest or use a Kubernetes cronjob.

Hmm, saying "create a manifest or cronjob" is confusing because a manifest just means the user needs to define a Kubernetes resource that will house the registry scanner. 

I think probably the type of resource that makes the most sense for the registry scanner is either a job or cronjob. Job would run the registry scanner once. Cronjob would allow the user to re-run the registry scanner on a regular interval. 

So I think our advice here is just to define a Kubernetes job or cronjob to deploy the registry scanner into a Kubernetes cluster.

Reply:
Fixed as noted.